submodules/storage/variables.tf

```diff
@@ -56,3 +56,39 @@ variable "efs_kms_key" {
   type        = string
   default     = null
 }
+
+variable "efs_backup_vault_kms_key" {
```

Comment:
is this the id or arn?

Reply:
It's the arn. This is keeping the pattern with the other variables in Steve's PR, so I'll leave any discussion on the variable name for there (and make sure it matches here at the time of merging).